### PR TITLE
Changed if..else and indentation

### PR DESCRIPTION
**Pull request by ayltonsilva**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -6,12 +6,12 @@ def hello
...
-        def logged_in_user 
-            unless logged_in?
-                store_location
-                flash[:danger] = "Please log in." 
-                redirect_to login_url
-            end 
-        end
+    # Confirms a logged-in user.
+    def logged_in_user 
+        unless logged_in?
```
Comment (marcospiridon) [inline app/controllers/application_controller.rb:11]: This is one more example where you could reduce lines.
Yes I know, I am being paranoiac.

```
return if logged_in?
store_location
flash[:danger] = "Please log in." 
redirect_to login_url
```

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -3,10 +3,7 @@ module ApplicationHelper
     #Returns the full title on a per-page basis.
     def full_title(page_title = '')
         base_title = "Ruby on Rails Tutorial Sample App"
-        if page_title.empty?
-            base_title
-        else
-            page_title + " | " + base_title
-        end
+        return base_title if page_title.empty?
```
Comment (marcospiridon) [inline app/helpers/application_helper.rb:6]: 👍

File: app/helpers/sessions_helper.rb [hunk 3]
```diff
@@ -19,7 +19,7 @@ def current_user?(user)
         
     # Returns true if the user is logged in, false otherwise.
     def logged_in? 
-        !current_user.nil?
+        current_user.present?
```
Comment (marcospiridon) [inline app/helpers/sessions_helper.rb:22]: 👍

File: app/helpers/sessions_helper.rb [hunk 4]
```diff
@@ -36,6 +36,6 @@ def redirect_back_or(default)
     
     # Stores the URL trying to be accessed.
     def store_location
-    session[:forwarding_url] = request.original_url if request.get?
+        session[:forwarding_url] = request.original_url if request.get?
```
Comment (marcospiridon) [inline app/helpers/sessions_helper.rb:39]: 2 spaces

**Review by marcospiridon — COMMENTED** [review]
Nice! 

I think there are some VSCode add ons to fix the indentation in several files at once. But dont care about that, as long as you get it.

Left a couple of comments

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (ayltonsilva) [reply app/controllers/application_controller.rb:11]: It's okay, I didn't except "return if" to be a better alternative for "unless"

**Pull request merged**